components/admin_console/billing/billing_subscriptions/limits.tsx

```diff
@@ -5,22 +5,33 @@ import React from 'react';
 import {useIntl, FormattedMessage} from 'react-intl';
 import {useDispatch, useSelector} from 'react-redux';
 
-import {cloudFreeEnabled} from 'mattermost-redux/selectors/entities/preferences';
 import {
     getCloudProducts,
     getCloudSubscription,
+    getSubscriptionProduct,
 } from 'mattermost-redux/selectors/entities/cloud';
+import {cloudFreeEnabled} from 'mattermost-redux/selectors/entities/preferences';
 
 import {openModal, closeModal} from 'actions/views/modals';
-import {asGBString} from 'utils/limits';
-import {ModalIdentifiers} from 'utils/constants';
-import CloudUsageModal from 'components/cloud_usage_modal';
-import MiniModal from 'components/cloud_usage_modal/mini_modal';
+import {trackEvent} from 'actions/telemetry_actions';
+
+import {ModalIdentifiers, CloudProducts} from 'utils/constants';
+import {FileSizes} from 'utils/file_utils';
+import {t} from 'utils/i18n';
+import {asGBString, fallbackStarterLimits} from 'utils/limits';
+
 import useGetLimits from 'components/common/hooks/useGetLimits';
 import useGetUsage from 'components/common/hooks/useGetUsage';
-import {t} from 'utils/i18n';
+import useOpenSalesLink from 'components/common/hooks/useOpenSalesLink';
+
+import CloudUsageModal from 'components/cloud_usage_modal';
+import MiniModal from 'components/cloud_usage_modal/mini_modal';
+import PricingModal from 'components/pricing_modal';
 
 import LimitCard from './limit_card';
+import SingleInlineLimit from './single_inline_limit';
+
+import './limits.scss';
 
 // TODO: To be removed once components can be used from where they can belong,
```

Comment:
Should we remove these now? The modals have been reviewed and we can reference the PR to see how to use them

Reply:
Sounds good to me.